<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../Users/luis.moraes/Desktop/ServentApp/Servent/Icons/construction_firm.ico</normaloff>../../../../../../Users/luis.moraes/Desktop/ServentApp/Servent/Icons/construction_firm.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="pagRequestMaterials">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>PEDIDO DE MATERIAIS</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>89</y>
        <width>411</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>ADICIONAR NOVO PEDIDO DE MATERIAIS</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QCommandLinkButton" name="btnLinkAddProvider">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Adicionar ?</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>249</y>
         <width>213</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Fornecedor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbProvider">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(255, 0, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>219</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Data:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="txtDate">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>5</day>
           </datetime>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>189</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Quantidade:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtQtd">
          <property name="inputMask">
           <string>0000;_</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>159</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Preço:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtPrice">
          <property name="inputMask">
           <string>00.00</string>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignJustify</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="cursorMoveStyle">
           <enum>Qt::LogicalMoveStyle</enum>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>51</x>
         <y>129</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Tipo de unidade:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtUnitType">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>51</x>
         <y>99</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMaterialName">
          <property name="inputMask">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>51</x>
         <y>70</y>
         <width>212</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Material:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbMaterial">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Selecione um tipo de material, ou então preencha os campos a baixo para adicionar um novo.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Os pedidos de materiais para a obra, são requeridos nessa pagina, e posteriormente enviados ao servidor e ao fornecedor. O historico total de pedido de materiais da obra, pode ser visto na pagina de &amp;quot;VISUALIZAÇÃO DAS MOVIMENTAÇÕES DE MATERIAIS&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>401</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lblInformation">
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnAddRequest">
           <property name="text">
            <string>ADICIONAR</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>LIMPAR CAMPOS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageCadProviders">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>CADASTRO DE FORNECEDORES</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>ADICIONAR NOVO FORNECEDOR</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>212</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Telefone:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txbPhone">
          <property name="inputMask">
           <string>(00) 000-000;_</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Rua:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txbStreet">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Número:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txbNumber">
          <property name="inputMask">
           <string>000000</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Nome: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txbProviderName">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Os cadastros de fornecedores serão efetuados atravéz dessa pagina, para visulizar todos os fornecedores já cadastrados acesse &amp;quot;Menu-&amp;gt;Visualizações-&amp;gt; Fornecedores&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>401</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblInfoProvider">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddProvider">
           <property name="text">
            <string>ADICIONAR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="brnClearFieldProvider">
           <property name="text">
            <string>LIMPAR CAMPOS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>450</height>
      </rect>
     </property>
     <attribute name="label">
      <string>VISUALIZAÇÃO DAS MOVIMENTAÇÕES DE MATERIAIS</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>441</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>421</width>
         <height>291</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Material</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fornecedor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Quantidade</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtMaterialName</tabstop>
  <tabstop>txtUnitType</tabstop>
  <tabstop>txtPrice</tabstop>
  <tabstop>txtQtd</tabstop>
  <tabstop>txtDate</tabstop>
  <tabstop>cmbProvider</tabstop>
  <tabstop>btnAddRequest</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnLinkAddProvider</tabstop>
  <tabstop>txbProviderName</tabstop>
  <tabstop>txbPhone</tabstop>
  <tabstop>txbStreet</tabstop>
  <tabstop>txbNumber</tabstop>
  <tabstop>btnAddProvider</tabstop>
  <tabstop>brnClearFieldProvider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>brnClearFieldProvider</sender>
   <signal>clicked()</signal>
   <receiver>txbNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brnClearFieldProvider</sender>
   <signal>clicked()</signal>
   <receiver>txbStreet</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brnClearFieldProvider</sender>
   <signal>clicked()</signal>
   <receiver>txbPhone</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brnClearFieldProvider</sender>
   <signal>clicked()</signal>
   <receiver>txbProviderName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtQtd</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtPrice</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtUnitType</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtMaterialName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
